index.bs

```diff
@@ -70,21 +70,28 @@ spec:generic-sensor-1;
 Introduction {#intro}
 ============
 
-The DeviceOrientationSensor API extends the Generic Sensor API [[GENERIC-SENSOR]]
-to provide information that describes the device's physical orientation in a stationary
-three dimensional Cartesian coordinate system.
+The OrientationSensor API extends the Generic Sensor API [[GENERIC-SENSOR]]
+to provide information that describes the device's physical orientation.
+
+The base object describes the orientation in relation to the earths magnetic
```

Comment:
The base object describes the rotation in relation to <a>Earth's reference coordinate system</a>?

"Earth's reference coordinate system" is already defined in model section, so we can ref it. Also "local coordinate system" is defined term that we can refer to.